RELEASE_NOTES

```diff
@@ -1,3 +1,10 @@
+========================================================================
+Amazon CloudWatch Agent 1.300044.0 (2024-08-14)
+========================================================================
+Bug Fixes:
+* [ContainerInsights] Update GPU usage metrics emitted 
+* [ContainerInsights] Deprecate runtime tag from neuron metrics to fix false average calculation 
```

Comment:
Should also mention this PR - https://github.com/aws/amazon-cloudwatch-agent/pull/1262